pyvista/plotting/renderer.py

```diff
@@ -1389,22 +1732,36 @@ def reset_camera(self, render=True, bounds=None):
             self.parent.render()
         self.Modified()
 
-    def isometric_view(self):
+    def view_isometric(self, negative=False):
         """Reset the camera to a default isometric view.
 
-        DEPRECATED: Please use ``view_isometric``.
```

Comment:
Similar to above, this PR is also removing functionality that is deprecated.  It may be best in a separate PR.